### DISCUSSION TITLE
Where is the 4GB iso to install to access the SOC

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5608] [Unsupported Versions] [answered]
Hi, i see there is an iso somewhere that requires only 4gb for first time users, but i cannot find it?
  I am using VMware workstation 16 pro.

**Comment by Sahale** [comment]
Hello!
Minimum hardware requirements for elastic is 8 GB, so you can not install full SO on system with 4 GB. 4 GB is minimum for import node according to [this](https://docs.securityonion.net/en/2.3/hardware.html)

**Comment by dougburks** [comment, marked as answer]
As described at https://docs.securityonion.net/en/2.3/first-time-users.html, you can use our standard ISO image and configure for IMPORT mode which will only require 4GB RAM in your VM. You can even install a desktop environment inside that 4GB VM which might make things easier for you.

**Comment by iqworks** [comment]
Thanks again Doug, I will be trying that here shortly. Do you happen to have
a link that explains

What having the extra 12GB of RAM for security onion is for? 

 

From: Doug Burks ***@***.***> 
Sent: Wednesday, September 22, 2021 1:46 PM
To: Security-Onion-Solutions/securityonion
***@***.***>
Cc: iqworks ***@***.***>; Author ***@***.***>
Subject: Re: [Security-Onion-Solutions/securityonion] Where is the 4GB iso
to install to access the SOC (Discussion #5608)

 

As described at https://docs.securityonion.net/en/2.3/first-time-users.html,
you can use our standard ISO image and configure for IMPORT mode which will
only require 4GB RAM in your VM. You can even install a desktop environment
inside that 4GB VM which might make things easier for you.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Security-Onion-Solutions/securityonion/discussions/5608#
discussioncomment-1371746> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABKPV35KHF744AZWHYZPAJTUD
I6BTANCNFSM5EPAQODQ> .
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&m
t=8&pt=524675>  or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=u
tm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
.
<https://github.com/notifications/beacon/ABKPV32357EI23FF5LJPCSTUDI6BTA5CNFS
M5EPAQOD2YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZG
OAAKO4YQ.gif> 



-- 
This email has been checked for viruses by AVG.
https://www.avg.com

**Reply by dougburks** [reply]
As described at https://docs.securityonion.net/en/2.3/first-time-users.html, IMPORT mode allows you to run so-import-pcap and import one or more pcap files.  It does not support sniffing live network traffic, so it can run in only 4GB RAM. However, it's still a great way to get experience with Security Onion! For more information, please see:
https://docs.securityonion.net/en/2.3/first-time-users.html
https://docs.securityonion.net/en/2.3/so-import-pcap.html
https://docs.securityonion.net/en/2.3/pcaps.html
https://docs.securityonion.net/en/2.3/analyst-vm.html

Once you have experience with analyzing pcaps in Security Onion and are ready to sniff live network traffic, you will need more resources as analyzing live traffic requires more processes and thus more RAM.  For more information, please see:
https://docs.securityonion.net/en/2.3/architecture.html
https://docs.securityonion.net/en/2.3/hardware.html
https://docs.securityonion.net/en/2.3/configuration.html
https://docs.securityonion.net/en/2.3/ingest.html

**Comment by iqworks** [comment]
Hi Doug, this is more great information! I will be looking at these links
this A.M. and creating a 4GB VM. 

Thanks tremendously 

From: Doug Burks ***@***.***> 
Sent: Thursday, September 23, 2021 3:09 AM
To: Security-Onion-Solutions/securityonion
***@***.***>
Cc: iqworks ***@***.***>; Author ***@***.***>
Subject: Re: [Security-Onion-Solutions/securityonion] Where is the 4GB iso
to install to access the SOC (Discussion #5608)

 

As described at https://docs.securityonion.net/en/2.3/first-time-users.html,
IMPORT mode allows you to run so-import-pcap and import one or more pcap
files. It does not support sniffing live network traffic, so it can run in
only 4GB RAM. However, it's still a great way to get experience with
Security Onion! For more information, please see:
https://docs.securityonion.net/en/2.3/first-time-users.html
https://docs.securityonion.net/en/2.3/so-import-pcap.html
https://docs.securityonion.net/en/2.3/pcaps.html
https://docs.securityonion.net/en/2.3/analyst-vm.html

Once you have experience with analyzing pcaps in Security Onion and are
ready to sniff live network traffic, you will need more resources as
analyzing live traffic requires more processes and thus more RAM. For more
information, please see:
https://docs.securityonion.net/en/2.3/architecture.html
https://docs.securityonion.net/en/2.3/hardware.html
https://docs.securityonion.net/en/2.3/configuration.html
https://docs.securityonion.net/en/2.3/ingest.html

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Security-Onion-Solutions/securityonion/discussions/5608#
discussioncomment-1373890> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABKPV37XMZO62YU6FJVEIZ3UD
L4B3ANCNFSM5EPAQODQ> .
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&m
t=8&pt=524675>  or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=u
tm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
.
<https://github.com/notifications/beacon/ABKPV32VEOXKLRI6UPFSWO3UDL4B3A5CNFS
M5EPAQOD2YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZG
OAAKPNQQ.gif> 



-- 
This email has been checked for viruses by AVG.
https://www.avg.com

**Comment by iqworks** [comment]
Hi Doug, 

   The good news is that using import mode allowed me to get an install that
worked!

 
![image](https://user-images.githubusercontent.com/5569263/134740292-30d18afa-3b66-4f6d-ad07-9acb3698bce0.png)

![image](https://user-images.githubusercontent.com/5569263/134740309-a61c0971-3084-4e7a-9e2a-405be8cdce39.png)
![image](https://user-images.githubusercontent.com/5569263/134740322-e3c5f88f-66b1-4a52-80a0-4cfe7f822b27.png)

![image](https://user-images.githubusercontent.com/5569263/134740333-01e39d56-5dc3-41a8-bfc5-bacdf528eae3.png)
![image](https://user-images.githubusercontent.com/5569263/134740351-79ca41ee-2539-4157-ba3e-0cd66ee788a5.png)

![image](https://user-images.githubusercontent.com/5569263/134740374-2c8a424e-d609-4e09-91be-71326d89bce0.png)

 
![image](https://user-images.githubusercontent.com/5569263/134740387-d586b1d5-5295-4cac-abbc-a86e2a48a8a1.png)


 

  But when I try to call up https://192.168.80.1 I get access denied??

 

 

From: Doug Burks ***@***.***>   
Sent: Thursday, September 23, 2021 3:09 AM
To: Security-Onion-Solutions/securityonion
***@***.***>
Cc: iqworks ***@***.***>; Author ***@***.***>
Subject: Re: [Security-Onion-Solutions/securityonion] Where is the 4GB iso
to install to access the SOC (Discussion #5608)

 

As described at https://docs.securityonion.net/en/2.3/first-time-users.html,
IMPORT mode allows you to run so-import-pcap and import one or more pcap
files. It does not support sniffing live network traffic, so it can run in
only 4GB RAM. However, it's still a great way to get experience with
Security Onion! For more information, please see:
https://docs.securityonion.net/en/2.3/first-time-users.html
https://docs.securityonion.net/en/2.3/so-import-pcap.html
https://docs.securityonion.net/en/2.3/pcaps.html
https://docs.securityonion.net/en/2.3/analyst-vm.html

Once you have experience with analyzing pcaps in Security Onion and are
ready to sniff live network traffic, you will need more resources as
analyzing live traffic requires more processes and thus more RAM. For more
information, please see:
https://docs.securityonion.net/en/2.3/architecture.html
https://docs.securityonion.net/en/2.3/hardware.html
https://docs.securityonion.net/en/2.3/configuration.html
https://docs.securityonion.net/en/2.3/ingest.html

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Security-Onion-Solutions/securityonion/discussions/5608#
discussioncomment-1373890> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABKPV37XMZO62YU6FJVEIZ3UD
L4B3ANCNFSM5EPAQODQ> .
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&m
t=8&pt=524675>  or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=u
tm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
.
<https://github.com/notifications/beacon/ABKPV32VEOXKLRI6UPFSWO3UDL4B3A5CNFS
M5EPAQOD2YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZG
OAAKPNQQ.gif> 



-- 
This email has been checked for viruses by AVG.
https://www.avg.com

**Reply by iqworks** [reply]
Hi Doug, I think i messed this post up a bit. I tried to cut and paste some images but they didnt take, so, i will try to organize things a little better here:

first, i did successfully install the import mode. I did use 200GB of storage, 4GB of RAM. 
Here are the IP values (and where I got them from) to fill out the rest. 
![image](https://user-images.githubusercontent.com/5569263/134741451-11668963-8085-4b9c-aa1e-ebd97d50a24a.png)
![image](https://user-images.githubusercontent.com/5569263/134741472-eb5ee555-ca1c-4b80-8167-fe7676e45341.png)
![image](https://user-images.githubusercontent.com/5569263/134741489-1895e221-661f-4727-8a94-6047471abb0b.png)
![image](https://user-images.githubusercontent.com/5569263/134741507-00511690-9304-4b15-a47e-fde8793f5317.png)
![image](https://user-images.githubusercontent.com/5569263/134741533-19c28b73-435f-4206-b1cc-5e9aa4ec3341.png)
![image](https://user-images.githubusercontent.com/5569263/134741559-77726059-4d94-4319-9dd8-856aae331813.png)
however, when i try https://192.168.80.1 i get access denied??

**Reply by dougburks** [reply]
If Windows is already using 192.168.80.1, then you can't use that same IP address in your VM.
